city_scrapers/spiders/il_sports_facilities_authority.py

```diff
@@ -0,0 +1,89 @@
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlSportsFacilitiesAuthoritySpider(CityScrapersSpider):
+    name = "il_sports_facilities_authority"
+    agency = "Illinois Sports Facilities Authority"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.isfauthority.com"]
+    start_urls = ["https://www.isfauthority.com/governance/board-meetings/"]
+    location = {
+        'name': 'Authority offices',
+        'address': 'Guaranteed Rate Field, 333 West 35th Street, Chicago, IL'
+    }
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        items = []
+        for item in response.css('div.wpb_text_column div.wpb_wrapper div.inner-text h2'):
+            items.append(item)
+        for item in response.css('div.wpb_text_column div.wpb_wrapper p'):
+            items.append(item)
+
+        for item in items:
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description='',
+                classification=BOARD,
+                start=self._parse_start(item),
+                end=None,
+                all_day=False,
+                time_notes='See agenda for meeting time',
+                location=self.location,
+                links=self._parse_links(item),
+                source=response.url,
+            )
+            try:
+                meeting['status'] = self._get_status(meeting)
+                meeting['id'] = self._get_id(meeting)
+            except TypeError:
+                continue
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse meeting title."""
+        title = item.css('::text').re_first(r'.*[a-zA-Z] Meeting')
+        if title and 'Board' in title:
+            return "Board of Directors"
+        return title
+
+    def _parse_location(self, item):
```

Comment:
Looks like this can be removed if it isn't present, but we'll still need a `_validate_location` function so that we can flag when the address might have changed

Reply:
pushed changes but didn't have time to finish last time. Now it's done :)